events/events.go

```diff
@@ -176,7 +176,7 @@ type ContainerLaunchContext struct {
 	Name                 string // used as the docker network name and part of container name. For microservice it is the ms instance key
 	AgreementId          string
 	Microservices        []MicroserviceSpec                     // Service dependencies go here. Microservices (in the workload/microservice model) never have dependencies.
-	ServicePathElement   persistence.ServiceInstancePathElement // The service that we're trying to start.
+	ServicePathElement   *persistence.ServiceInstancePathElement // The service that we're trying to start.
```

Comment:
Please do not change it to pointer because pointer in the event structure may cause segmentation fault where passing from a thread to another.